src/react-reusable.js

```diff
@@ -2,7 +2,7 @@ import React, { useState, useContext, useEffect } from "react";
 import {shallowEqual} from './shallow-equal';
 import {getStore} from './reusable';
 
-export const ReusableContext = React.createContext();
+const ReusableContext = React.createContext();
```

Comment:
I'd either leave this or let the user provide a store in ReusableProvider (to allow for server side rendering)

Reply:
That's why I exported replaceStore and createStore. Because in context it's already too late (reusable access the default store, not via context)